crates/uv/src/commands/pip_show.rs

```diff
@@ -0,0 +1,139 @@
+use std::fmt::Write;
+
+use anyhow::Result;
+use owo_colors::OwoColorize;
+use tracing::debug;
+
+use distribution_types::Name;
+use platform_host::Platform;
+use uv_cache::Cache;
+use uv_fs::Simplified;
+use uv_installer::SitePackages;
+use uv_interpreter::PythonEnvironment;
+use uv_normalize::PackageName;
+
+use crate::commands::ExitStatus;
+use crate::printer::Printer;
+
+/// Show information about one or more installed packages.
+pub(crate) fn pip_show(
+    mut packages: Vec<PackageName>,
```

Comment:
@ChannyClaus - I simplified things a bit here because this method only needs to accept a list of package names (unlike `uninstall` which needs to accept a list of requirements files, etc.).